monkey/common/types.py

```diff
@@ -17,6 +17,16 @@
 HardwareID: TypeAlias = PositiveInt
 MachineID: TypeAlias = PositiveInt
 
+JSONSerializable = Union[  # type: ignore[misc]
+    Dict[str, "JSONSerializable"],  # type: ignore[misc]
+    List["JSONSerializable"],  # type: ignore[misc]
```

Comment:
If we upgrade mypy to 0.981 we can avoid these errors using `--enable-recursive-aliases` flag.

Reply:
Can you push updated lock files?